emscripten.py

```diff
@@ -2360,6 +2360,28 @@ def create_asm_consts_wasm(forwarded_json, metadata):
       all_sigs.append((sig, call_type))
 
   asm_const_funcs = []
+  asm_const_funcs.append(r'''
+function _read_asm_const_args(sig_ptr, buf) {
+  var args = [];
+  var sig = AsciiToString(sig_ptr);
+  function align_to(ptr, align) {
```

Comment:
`align_to` should also be `alignTo` but I understand that's a pre-existing issue...

Reply:
I mean it's a pre-existing issue in the code I'd written earlier and in-line with the other snake_case -> camelCase renames so I'm fine with that <.<